README.md

```diff
@@ -55,3 +59,10 @@ The SSH tunnel can then be used for things like connecting to an RDS database th
 
 `aws-connect -s -t CLUSTER=prod`
 
+5. Run SSM Document named shell-script on shopify with default profile and arguments 'param1 param 2'. The cloudwatch log name has been changed to ssm-cloudwatch-logs. Document is required, github token is only required for private repos: 
+
+`aws-connect -x instance -r region -p default -a document -d shell-script -p default -w 'param1 "param 2"' -g /devops/github_token -c ssm-cloudwatch-logs` 
```

Comment:
Use real (made up) values here since it's meant to be an actual example command.  For example

`aws-connect -x i-1234567890 -r us-east-1 -p staging -a document -d my-ssm-document -w 'param1 "param 2"' -g /devops/github_token -c ssm-cloudwatch-logs`